Updating devices after an ESPHome 2024.9.2

Hi, I got an error when trying to update raspi-audio. Seems mode: as to be replaced by channel:
Cordialy

-Christian

Updating /config/esphome/raspiaudio-muse-proto-381dec.yaml
------------------------------------------------------------

INFO ESPHome 2024.9.0
INFO Reading configuration /config/esphome/raspiaudio-muse-proto-381dec.yaml...
INFO Updating https://github.com/esphome/firmware.git@main
INFO Updating https://github.com/esphome/media-players.git@main
WARNING GPIO5 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
Failed config

speaker.i2s_audio: [source /data/packages/7b8574f6/raspiaudio/raspiaudio-muse-proto.yaml:38]
  platform: i2s_audio
  id: board_external_speakers
  dac_type: external
  i2s_dout_pin: 
    number: GPIO26
    allow_other_uses: True
  
  [mode] is an invalid option for [speaker.i2s_audio]. Did you mean [i2s_mode]?
  mode: mono
 [ERROR] /config/esphome/raspiaudio-muse-proto-381dec.yaml

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -24,7 +24,7 @@ jobs:
         onju-voice/onju-voice.factory.yaml
         raspiaudio/raspiaudio-muse-luxe.factory.yaml
         raspiaudio/raspiaudio-muse-proto.factory.yaml
-      esphome-version: 2024.7.3
+      esphome-version: 2024.9.1
       release-summary: ${{ github.event_name == 'release' && github.event.release.body || '' }}
       release-url: ${{ github.event_name == 'release' && github.event.release.html_url || '' }}
       release-version: ${{ github.event_name == 'release' && github.event.release.tag_name || '' }}

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,6 @@
 # Gitignore settings for ESPHome
 # This is an example and may include too much for your use-case.
 # You can modify this file to suit your needs.
-/.esphome/
+.esphome/
 /secrets.yaml
 venv

--- a/raspiaudio/raspiaudio-muse-proto.yaml
+++ b/raspiaudio/raspiaudio-muse-proto.yaml
@@ -1,7 +1,7 @@
 esphome:
   name: raspiaudio-muse-proto
   friendly_name: RaspiAudio Muse Proto
-  min_version: 2024.6.0
+  min_version: 2024.9.0
   name_add_mac_suffix: true
 
 esp32:
@@ -41,7 +41,7 @@ speaker:
     i2s_dout_pin:
       number: GPIO26
       allow_other_uses: true
-    mode: mono
+    channel: mono
 
 media_player:
   - platform: i2s_audio

